src/pymodaq/daq_utils/managers/modules_manager.py

```diff
@@ -38,15 +99,19 @@ class ModulesManager(QObject):
         ]},
     ]
 
-    def __init__(self, detectors=[], actuators=[], selected_detectors=[], selected_actuators=[], timeout=10000):
+    def __init__(self, detectors=[], actuators=[], selected_detectors=[], selected_actuators=[], timeout_s=None):
```

Comment:
Mutable objects as default parameters should be avoided (see [here](https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/)).

The recommended alternative is:

```python
def __init__(
    self,
    detectors=None,
    actuators=None,
    selected_detectors=None,
    selected_actuators=None,
    timeout_s=None,
):
    if detectors is None:
        detectors = []
    if actuators is None:
        actuators = []
    if selected_detectors is None:
        selected_detectors = []
    if selected_actuators is None:
        selected_actuators = []
```

Or:

```python
detectors = detectors or []
actuators = actuators or []
# ...
```

Reply:
ok you can do it